ocaml/driver/main_args.ml

```diff
@@ -716,11 +716,18 @@ let mk_dump_into_file f =
 ;;
 
 let mk_extension f =
-  "-extension", Arg.String f, "<extension> Enable the extension"
+  let available_extensions =
+    Clflags.Extension.(List.map to_string all)
+  in
+  "-extension", Arg.Symbol (available_extensions, f),
+  "<extension> Enable the extension (may be specified more than once)"
```

Comment:
The surrounding code usually has two spaces in such places.
```suggestion
  "<extension>  Enable the extension (may be specified more than once)"
```

Reply:
Perhaps this wording makes it clearer that the flag takes an argument?

```suggestion
  "<extension> Enable the given extension (may be specified more than once)"
```